src/Swarm/TUI/Controller.hs

```diff
@@ -937,6 +938,56 @@ handleREPLEventPiloting x = case x of
       & replPromptText .~ nt
       & replPromptType .~ CmdPrompt []
 
+runBaseWebCode :: MonadState AppState m => T.Text -> m String
```

Comment:
At first the return value was `m ()`, but I made it return a `String` for debugging purposes.